integration_tests/graalvm_tests/src/main/resources/META-INF/native-image/org.apache.fury/graalvm_tests/native-image.properties

```diff
@@ -24,4 +24,7 @@ Args = -H:+ReportExceptionStackTraces \
   org.apache.fury.graalvm.ThreadSafeExample,\
   org.apache.fury.graalvm.ProxyExample,\
   org.apache.fury.graalvm.Benchmark,\
-  org.apache.fury.resolver.DisallowedList
+  org.apache.fury.resolver.DisallowedList,\
```

Comment:
Could we add those lines in `java/fury-core/src/main/resources/META-INF/native-image/org.apache.fury/fury-core/native-image.properties` instead